src/sentry/api/serializers/models/group.py

```diff
@@ -422,7 +422,9 @@ def _get_status(self, attrs, obj):
                         "ignoreUntil": snooze.until,
                         "ignoreUserCount": (
                             snooze.user_count - (attrs["user_count"] - snooze.state["users_seen"])
-                            if snooze.user_count and not snooze.user_window
+                            if snooze.user_count
+                            and not snooze.user_window
+                            and not self._collapse("stats")
```

Comment:
will we also need to add a similar check to line 418 for when we try to access obj.times_seen? Or is times_seen available when stats are collapsed?

Reply:
`times_seen` is always available since it's a property on the group itself